packages/shared/src/components/sidebar/MyFeedButton.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { ReactElement, useContext } from 'react';
+import classNames from 'classnames';
+import PlusIcon from '@dailydotdev/shared/icons/plus.svg';
+import FilterIcon from '@dailydotdev/shared/icons/outline/filter.svg';
+import { IFlags } from 'flagsmith';
+import { Button } from '../buttons/Button';
+import FeaturesContext from '../../contexts/FeaturesContext';
+import { Features, getFeatureValue } from '../../lib/featureManagement';
+import { ButtonOrLink, ItemInner, NavItem, SidebarMenuItem } from './common';
+
+const statusColor = {
+  success: {
+    border: 'border-theme-status-success',
+    shadow: 'shadow-2-avocado',
+    button: 'btn-primary-avocado',
+  },
+  error: {
+    border: 'border-theme-status-error',
+    shadow: 'shadow-2-ketchup',
+    button: 'btn-primary-ketchup',
+  },
+  help: {
+    border: 'border-theme-status-help',
+    shadow: 'shadow-2-cheese',
+    button: 'btn-primary-cheese',
+  },
+  warning: {
+    border: 'border-theme-status-warning',
+    shadow: 'shadow-2-bun',
+    button: 'btn-primary-bun',
+  },
+  cabbage: {
+    border: 'border-theme-status-cabbage',
+    shadow: 'shadow-2-cabbage',
+    button: 'btn-primary-cabbage',
+  },
+  fill: {
+    border: 'border-theme-status-fill',
+    shadow: 'shadow-2-water',
+    button: 'btn-primary-water',
+  },
+};
+
+const UnfilteredMyFeedButton = ({
+  sidebarExpanded,
+  flags,
+  action,
+}: {
+  sidebarExpanded: boolean;
+  flags: IFlags;
+  action: () => unknown;
+}) => {
+  const buttonCopy =
+    getFeatureValue(Features.MyFeedButtonCopy, flags) || 'Create my feed';
```

Comment:
I think it's better to add a default value support for `getFeatureValue`, it would look much nicer. What do you think?

Reply:
Great point, i'll look into this 👏